src/client/components/home/index.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import MatchResults from 'client/components/matchResults';
 
 const Home = () => (
-  <div id="home-wrapper">
+  <div data-testid="home-wrapper" id="home-wrapper">
```

Comment:
Wooh! What is the `data-testid` used for 👀 ? Just curious about that 😂

Reply:
This is for testing,  `@testing-library` has APIs to do dom testing
ref: https://testing-library.com/docs/dom-testing-library/api-queries#bytestid

Reply:
We can remove `data-testId` by using [babel-plugin](https://www.npmjs.com/package/babel-plugin-jsx-remove-data-test-id) if we don't want to leave it in `production` dom